src/components/ApplicationsMenu.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div v-if="!!applicationsList.length">
+    <oc-button id="_appSwitcherButton" icon="menu" variation="primary" class="oc-topbar-menu-burger uk-height-1-1"  aria-label="$gettext('Application Switcher')" ref="menubutton" />
+    <oc-drop toggle="#_appSwitcherButton" mode="click" :options="{pos:'bottom-right'}" class="uk-width-large" ref="menu">
+      <div class="uk-grid-small uk-text-center" uk-grid>
+        <div class="uk-width-1-3" v-for="(n, nid) in applicationsList" :key="nid">
+          <a :href="n.route ? n.route.path : null" @click="openItem(n.url)">
+            <oc-icon :name="n.iconMaterial" size="large" />
+            <div>{{ translateMenu(n) }}</div>
+          </a>
+        </div>
+      </div>
+    </oc-drop>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    visible: {
+      type: Boolean,
+      required: false,
+      default: false
+    },
+    applicationsList: {
+      type: Array,
+      required: false,
+      default: () => null
+    }
+  },
+  watch: {
+    visible (val) {
+      if (val) {
+        this.focusFirstLink()
+      } else {
+        this.$emit('closed')
+      }
+    }
+  },
+  computed: {
+    _logoutItemText () {
+      // return this.$gettextInterpolate(this.$gettext('Exit %{product}'), { product: this.configuration.theme.general.name })
+      return 'Logout' // TODO
+    }
+  },
+  methods: {
+    logout () {
+      this.visible = false
+      this.$store.dispatch('logout')
+    },
+    translateMenu (navItem) {
+      // FIXME need to know the locale
+      // return this.$gettext(navItem.name)
+      return navItem.name
+    },
+    openItem (url) {
+      if (url) {
+        const win = window.open(url, '_blank')
+        win.focus()
```

Comment:
please only call focus if win is defined - see https://github.com/owncloud/phoenix/pull/2651/files#diff-b53adc56b49b58de3849fd52d310f967R200

there are cases where win is undefined - e.g. opening a tab is disallowed

Reply:
this was a copy-paste from an earlier compnent, somehow it didn't have this.

will add

Reply:
I've changed the approach to use `<a target='_blank'` which yields the same result